src/components/Form/Form.tsx

```diff
@@ -46,6 +48,14 @@ const Form = (props: FormProps) => {
 
 	useEffect(() => {
 		setTopComponentHeight(topComponentRef.current?.offsetHeight);
+		setTopComponentDrawerHeight && setTopComponentDrawerHeight(`${topComponentRef.current?.offsetHeight}px`);

```

Comment:
Several comments here:
1. Same comment as in FormTypes.tsx
2. This wouldn't always work as expected because formContentRef wasn't added to the dependency array. 

Once the context / reducer approach gets solved I believe all of this can be deleted and moved to where it's actually needed.